ui/components/Page.tsx

```diff
@@ -20,15 +20,13 @@ export type PageProps = {
 };
 
 export const Content = styled.div`
-  min-height: 85vh;
-  max-width: 1400px;
   margin: 0 auto;
   width: 100%;
   min-width: 1260px;
   box-sizing: border-box;
   background-color: rgba(255, 255, 255, 0.75);
   padding-left: ${(props) => props.theme.spacing.large};
-  padding-right: ${(props) => props.theme.spacing.large};
+
```

Comment:
Empty line here looks strange. Can we remove it, to move before `padding-left` to at least the padding rules are grouped?